subprojects/launcher/launcher.gradle.kts

```diff
@@ -77,8 +77,10 @@ dependencies {
     integTestRuntimeOnly(files(toolsJar))
 }
 
-tasks.jar {
-    val classpath = listOf(":bootstrap", ":baseServices", ":coreApi", ":core").joinToString(" ") {
+evaluationDependsOn(":persistentCache")
```

Comment:
Gah. Why is this needed?

Reply:
Because the `persistentCache` project is not configured at the time we access the `jar` task on it if I don't add it. I think the other tests work accidentally. I'll ask how we can handle this code without reaching into the other projects.

Reply:
Can't we use a provider here to configure the classpath attribute that would be evaluated at execution time insted?

Reply:
We should probably also use `locationOnly` instead of `get()`, no?

Reply:
I am now using a new configuration to configure the classpath. `locationOnly` only works on `FileSystemLocationProperty` and not on the providers I have here.